Frameworks/CoreText/CTFont.mm

```diff
@@ -101,25 +110,74 @@
 const CFStringRef kCTFontFeatureSelectorSettingKey = static_cast<CFStringRef>(@"kCTFontFeatureSelectorSettingKey");
 
 /**
+ All these values are fixed number in IOS no matter on iphone* or ipad*.
+ */
+const CGFloat kCTFontLabelFontSize = 17.0f;
+const CGFloat kCTFontButtonFontSize = 14.0f;
+const CGFloat kCTFontSmallSystemFontSize = 12.0f;
+const CGFloat kCTFontSystemFontSize = 14.0f;
+
+using namespace Microsoft::WRL;
+
+struct __CTFont {
```

Comment:
- Is the same style as CF
- For most purposes, since this is already def'd to CTFontRef, you can just use that
- The only times when 'struct' is used is when directly modifying the struct members, and I don't dislike the emphasis there

---
In reply to: [78220566](https://github.com/Microsoft/WinObjC/pull/987#discussion_r78220566) [](ancestors = 78220566,78205733)